modules/json/src/helpers/parse-expression.js

```diff
@@ -0,0 +1,65 @@
+import {get} from '../utils/get';
+
```

Comment:
Why is this renamed? Does it have anything to do with this feature?

Reply:
Because we are passing `MemberExpression` parsed types and `wrapperFn` as well, so at the end we are evaluating the data string (`row[prop]`) wrapped in a function, does it make sense?

Reply:
You are just adding an argument, not changing the fact that `propValue` is still a string. Undoing the renaming will significantly clean up this PR.

Reply:
Let's leave it like before then